packages/@expo/metro-config/src/transform-plugins/importExportLiveBindings.ts

```diff
@@ -289,8 +285,7 @@ export function importExportLiveBindingsPlugin({
         }
         const source: ModuleRequest = path.node.source;
         if (!path.node.specifiers.length) {
-          // NOTE(@kitten): The tree-shaking plugin leaves around empty specifiers, so we should
-          // handle this properly. This doesn't indicate a side-effect!
+          addImport(path, state, source);
```

Comment:
Now this indicates side effect, as we know this is coming from the original source (not our transformation -> tree shaking).